packages/openneuro-app/src/scripts/utils/gtag.js

```diff
@@ -6,7 +6,8 @@ function gtag() {
 
 gtag('js', new Date())
 
-export const initialize = trackingId => gtag('config', trackingId)
+export const initialize = trackingIds =>
+  trackingIds.forEach(trackingId => gtag('config', trackingId))
```

Comment:
This doesn't look like it will send all events to all tracking ids?

Reply:
`gtag('config', trackingId)` just needs to be called for each tracking ID, and subsequent calls will go to both unless specified with a `send_to` property.

https://www.optimizesmart.com/using-multiple-trackers-for-cross-domain-tracking-in-universal-analytics/#b8

Reply:
Okay, great. I found the current documentation for that.

https://developers.google.com/analytics/devguides/collection/gtagjs/#configure_additional_google_analytics_properties